src/NuGet.Clients/NuGet.PackageManagement.UI/Resources.resx

```diff
@@ -1005,4 +1005,20 @@ Please see https://aka.ms/troubleshoot_nuget_cache for more help.</value>
     <value>This package version is deprecated. Use {0} instead.</value>
     <comment>{0} is a NuGet package id (string)</comment>
   </data>
+  <data name="Billion" xml:space="preserve">
+    <value>{0:G3}B</value>
+    <comment>Billions in short format notation</comment>
```

Comment:
Please specify which billions are you referring to:

```suggestion
    <comment>American billions in short format notation</comment>
```

Reply:
Addressed.